automation_tools/__init__.py

```diff
@@ -1055,8 +1055,13 @@ def setup_foreman_discovery(sat_version):
     run('hammer -u admin -p {0} template dump --name '
         '"PXELinux global default" > {1}'.format(
             admin_password, template_file))
-    run('sed -i -e "s/^ONTIMEOUT\s\+local/ONTIMEOUT discovery/" {0}'
-        .format(template_file))
+    if sat_version == '6.3':
+        # since 6.4, ONTIMEOUT option uses "default_pxe_item_global" setting
+        run('sed -i -e "s/^ONTIMEOUT\s\+local/ONTIMEOUT discovery/" {0}'
+            .format(template_file))
+    else:
+        run('hammer -u admin -p {0} settings set --name '
```

Comment:
@rplevka value of {0} ?

Reply:
missed that during copy&pasta..great catch

Reply:
fixed. pls re-review